### PR TITLE
Fix token_type_ids error for big_bird model.

### PR DESCRIPTION
**Pull request by wlhgtc**
I run the follow code, but get strange outputs, the `token_type_ids` is shorter than `input_ids`:
```python
def demo():
    model_name = "./resources/bigbird-roberta-base"
    tokenizer = AutoTokenizer.from_pretrained(model_name)
    text = 'With power of science ,'
    max_length = 10
    encoded_tokens = tokenizer.encode_plus(
        text=text,
        add_special_tokens=True,
        max_length=max_length,
        truncation=True if max_length is not None else False,
        return_tensors=None,
        return_offsets_mapping=tokenizer.is_fast,
        return_attention_mask=False,
        return_token_type_ids=True,
        return_special_tokens_mask=True,
    )
    print(encoded_tokens)
```
CleanShot 2021-04-21 at 19 03 13@2x
Seem it miss the `create_token_type_ids_from_sequences` method, so it run code in [here](https://github.com/huggingface/transformers/blob/95dab34d5588fb155dfed8293ac2fbb1217a95a7/src/transformers/tokenization_utils_base.py#L2665) as default. So I copy the method in BERT to fix it.

I know maybe the big_bird model doesn't need `token_type_ids`, but we also have to make sure it return the right result if we set `return_token_type_ids = True`.

### CODE REVIEW

**Comment by wlhgtc** [comment]
@sgugger

**Review by sgugger — APPROVED** [review]
LGTM, but I'm no expert on BigBird so deferring the final decision to merge to @patrickvonplaten 

Thanks for the PR!

File: src/transformers/models/big_bird/tokenization_big_bird.py [hunk 1]
```diff
@@ -226,3 +226,27 @@ def get_special_tokens_mask(
         if token_ids_1 is None:
             return [1] + ([0] * len(token_ids_0)) + [1]
         return [1] + ([0] * len(token_ids_0)) + [1] + ([0] * len(token_ids_1)) + [1]
+
+    def create_token_type_ids_from_sequences(
+        self, token_ids_0: List[int], token_ids_1: Optional[List[int]] = None
+    ) -> List[int]:
+        """
+        Create a mask from the two sequences passed to be used in a sequence-pair classification task. A BERT sequence
```
Comment (patrickvonplaten) [inline src/transformers/models/big_bird/tokenization_big_bird.py:234]:
```suggestion
        Create a mask from the two sequences passed to be used in a sequence-pair classification task. A BigBird sequence
```

**Comment by patrickvonplaten** [comment]
This looks good to me! Also pinging @vasudevgupta7 here

**Comment by vasudevgupta7** [comment]
Yes, both base & large pre-trained checkpoints accept 2 token type ids while only trivia-qa checkpoint accepts 16 token type ids. So, this should be good.

**Pull request merged**